core/modules/qmeta/QMetaMysql.h

```diff
@@ -185,8 +224,8 @@ class QMetaMysql : public QMeta {
      *  @param queryId:   Query ID, non-negative number.
      *  @return: List of query IDs.
      */
-    virtual std::vector<int> getQueriesForTable(std::string const& dbName,
-                                                std::string const& tableName);
+    virtual std::vector<QueryId> getQueriesForTable(std::string const& dbName,
+                                                    std::string const& tableName);
 
 protected:
 
```

Comment:
General comment on QMetaMysql.h - I think you should get rid of the doxygen comment blocks for all virtual functions. doxygen will automatically copy in the docs from the method you are implementing (if the appropriate doxygen configuration parameter, INHERIT_DOCS is set to YES). If you'd rather make the copying explicit, you can use @copydoc.